### PR TITLE
chore: bump ethabi version

### PR DESCRIPTION
**Pull request by Genysys**
## Motivation

This PR bumps ethabi version required to close https://github.com/foundry-rs/foundry/issues/1432 . 
## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Updated the changelog

### CODE REVIEW

File: ethers-core/Cargo.toml [hunk 1]
```diff
@@ -50,7 +57,9 @@ hex-literal = "0.3.4"
 rand = "0.8.5"
 
 [features]
-celo = ["legacy"] # celo support extends the transaction format with extra fields
+celo = [
+    "legacy",
+] # celo support extends the transaction format with extra fields
```
Comment (mattsse) [inline ethers-core/Cargo.toml:62]: unrelated

File: ethers-core/Cargo.toml [hunk 2]
```diff
@@ -11,14 +11,21 @@ keywords = ["ethereum", "web3", "celo", "ethers"]
...
 
 # crypto
 elliptic-curve = { version = "0.11.12", default-features = false }
 generic-array = { version = "0.14.5", default-features = false }
-k256 = { version = "0.10.4", default-features = false, features = ["keccak256", "ecdsa", "std"] }
+k256 = { version = "0.10.4", default-features = false, features = [
+    "keccak256",
+    "ecdsa",
+    "std",
+] }
```
Comment (mattsse) [inline ethers-core/Cargo.toml:28]: unchanged?

**Review by mattsse — CHANGES_REQUESTED** [review]
ty!

undo line breaks for unrelated dependencies

File: ethers-core/Cargo.toml (hunk 1 above)
Comment (Genysys) [reply ethers-core/Cargo.toml]: done

File: ethers-core/Cargo.toml (hunk 2 above)
Comment (Genysys) [reply ethers-core/Cargo.toml]: done

**Review by mattsse — APPROVED** [review]

**Review by gakonst — APPROVED** [review]
